kodi_addon_checker/check_addon.py

```diff
@@ -384,3 +392,23 @@ def _check_dependencies(report: Report, addon_path, repo_addons):
             if LooseVersion(available_version) < LooseVersion(required_version) and (required_addon not in ignore):
                 report.add(Record(PROBLEM, "Version mismatch for addon %s. Required: %s, Available: %s "
                                   % (required_addon, required_version, available_version)))
+
+
+def get_addon_name(xml_path):
+    tree = ET.parse(xml_path).getroot()
+    return (tree.get("id"), tree.get("version"))
+
+
+def _check_for_existing_addon(report: Report, addon_xml, repo_addons, branch_name):
+    """Check if addon submitted already exists or not"""
+    addon_xml = os.path.join(addon_xml, "addon.xml")
+    addon_name, addon_version = get_addon_name(addon_xml)
+
+    if addon_name not in repo_addons:
+        report.add(Record(INFORMATION, "This is a new addon"))
+    else:
```

Comment:
We need another case here if the addon is in repo_addon and has the same version there as it has in the local addon.xml. In that case you need to Record a PROBLEM as we can't accept addons that are not version bumped.